tests/fixtures/parser/ftl/05-variants-errors.ftl

```diff
@@ -1,28 +1,37 @@
-key1 =
+key1 = {
  [:key] Value
+}
 
-key3 =
+key3 = {
  [] Value
+}
 
-key4 =
+key4 = {
  **[f] Foo
+}
 
-key5 =
+key5 = {
  *fw] Foo
+}
 
-key6 =
+key6 = {
   [ a] A
```

Comment:
Many of these examples will error simply because there's no default variant in the select expression. Consider adding a second valid one, e.g. `*[ok] Valid`.